log4j-core-test/src/test/resources/log4j-test3.xml

```diff
@@ -15,7 +15,7 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<Configuration status="OFF" name="XMLConfigTest" monitorInterval="5" shutdownHook="disable">
+<Configuration status="DEBUG" name="XMLConfigTest" monitorInterval="5" shutdownHook="disable">
```

Comment:
I would prefer to revert this or remove `status` altogether, so we don't pollute the build logs.

Reply:
Agreed with @ppkarwasz.

Reply:
Fixed.